project/kexuan5458/README.md

```diff
@@ -0,0 +1,64 @@
+# Self-Driving Car Localization
+
+## Basic Information
+
+Kalman Filter and Iterative Closest Point(ICP) algorithm are good tools in the self-driving car localization task.
```

Comment:
Which one will you focus on?  The Kalman filter or ICP, or they are coupled?

Reply:
They are coupled.